src/ColorGrid.js

```diff
@@ -11,6 +11,17 @@ const ColorGrid = ({ hueValue, saturationCount, luminosityCount }) => {
   const oneColorWidth = width / (saturationCount - 2) / 2;
   const oneColorHeight = height / (luminosityCount - 2) / 2;
 
+  const hslToHex = (h, s, l) => {
+    l /= 100;
+    const a = s * Math.min(l, 1 - l) / 100;
+    const f = n => {
+      const k = (n + h / 30) % 12;
+      const color = l - a * Math.max(Math.min(k - 3, 9 - k, 1), -1);
+      return Math.round(255 * color).toString(16).padStart(2, '0');
+    };
+    return `#${f(0)}${f(8)}${f(4)}`;
```

Comment:
Thanks for writing this logic, but there's an already included dependency that can do this, see if you can use it

import { hsl } from 'color-convert';
hsl.hex(h,s,l) //-> this will give you the HEX code directly.

other than that, looks great!

Reply:
Thanks, I fixed it now.